panther_bringup/config/imu_config.yaml

```diff
@@ -0,0 +1,44 @@
+imu_filter_node:
```

Comment:
`imu_config.yaml` or `imu.yaml`? Considering other config in this pkg and other Husarion robots utilize `ekf.yaml`, etc., which one would you prefer?